ingest/ingest_retry_client.go

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2023 IndyKite
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ingest
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"google.golang.org/grpc/codes"
+
+	"github.com/indykite/indykite-sdk-go/errors"
+	ingestpb "github.com/indykite/indykite-sdk-go/gen/indykite/ingest/v1beta2"
+	api "github.com/indykite/indykite-sdk-go/grpc"
+)
+
+type RetryPolicy struct {
+	InitialBackoff    string
```

Comment:
maybe some documentation on what values the `InitialBackoff` supports?

Reply:
Why it is not of type `time.Duration` already, when you parse it to time.Duration anyway?

Reply:
I based it on this policy suggested by grpc team. So want to keep as similar as possible. https://github.com/grpc/proposal/blob/master/A6-client-retries.md#retry-policy-capabilities

Reply:
They say, it must follow JSON structure of underlying Proto message. You have no underlying Proto nor JSON. So I would go with time.Duration directly.

Reply:
Agree to disagree 😄 
I'll keep it this way as it's only used when having to reconnect and should not affect perfomance